frontend/src/pages/GamemodesPage/GamemodesPage.tsx

```diff
@@ -15,13 +17,15 @@ function GamemodesPage() {
           cardTitleText="Exploration"
           cardBodyText="Explore UNSW’s lush campus at your own pace!"
           img={poggers}
+          onClick={() => { navigate('/play'), { state: { gamemode: 'exploration' } } }}
```

Comment:
confused about how this passes the state through

Reply:
https://reactrouter.com/en/main/hooks/use-navigate#optionsstate

Reply:
i fucked up the syntax tho ill fix

Reply:
> i fucked up the syntax tho ill fix

fixed in latest commit

Reply:
made changes in `game` branch to retrieve gamemode from this, ill resolve for now but pls msg if i can clarify at all